ci/deps/travis-37-cov.yaml

```diff
@@ -31,16 +29,18 @@ dependencies:
   - odfpy
   - openpyxl
   - pandas-gbq
+  - google-cloud-bigquery>=1.27.2 # GH 36436
   - psycopg2
   - pyarrow>=0.15.0
-  - pymysql
+  - pymysql=0.7.11
```

Comment:
It doesn't relate to gbq. Just trying to find out if the sql failures are due to new packages.

Reply:
I could revert this and open a new issue about sql. But the cov build would fail.